libs/features/georoutingv2/georoutingv2.js

```diff
@@ -110,14 +113,18 @@ function getGeoroutingOverride() {
   return hideGeorouting === 'on';
 }
 
-function decorateForOnLinkClick(link, prefix) {
+function decorateForOnLinkClick(link, prefix, currPrefix) {
   link.addEventListener('click', () => {
     const modPrefix = prefix || 'us';
+    const modCurrPrefix = currPrefix || 'us';
     // set cookie so legacy code on adobecom still works properly.
     const domain = window.location.host === 'adobe.com'
       || window.location.host.endsWith('.adobe.com') ? 'domain=adobe.com' : '';
     document.cookie = `international=${modPrefix};path=/;${domain}`;
     link.closest('.dialog-modal').dispatchEvent(new Event('closeModal'));
+    if (currPrefix !== undefined) {
```

Comment:
If `currPrefix` is `null` this condition is met. Could we instead check for `.length`?

Reply:
So this check is to to verify is currPrefix has been passed to the function then only do the next step. I have verified if the currPrefix (it is the third parameter in the function definition) is not passed in the function and it comes as undefined when not passed.

Also length cannot be used as for when we visit US page from say France prefix is passed as '' but we still want to call analytics page. We only don't want it to be called when it is not passed in the function

Reply:
could you then use undefined as a default param? 
`function decorateForOnLinkClick(link, prefix, currPrefix = undefined) {`

Reply:
It'll be undefined when not passed so was wondering if assigning the same default value will matter. Let me know if you think we should assign some other default value to it.